### DISCUSSION TITLE
Home Assistant Current Prices

### DISCUSSION THREAD

**Discussion by melvanderwal** [amberelectric/public-api#140] [Home Assistant]
Hi there.

Do the `*_feed_in_price` and `*_general_price` sensors from the HA integration adhere to this description from the API (and particularly the emphasised part)? 

> Returns the current interval's forecasted price comprised of the weighted average of 5-minute actual prices and 5-minute forecast prices. _**In the last 5-minutes of the interval, the price represents the final price for that interval.**_

And the follow-up question: just to confirm, the feed in price that's available for the last 5 minutes will be applied to all energy exported between the start time and end time?

**Comment by madpilot** [comment]
Correct.

**Comment by melvanderwal** [comment]
Thanks madpilot.  Hopefully I'll have something for show 'n tell shortly.
